cfme/rest/__init__.py

```diff
@@ -354,3 +355,23 @@ def mark_vm_as_template(rest_api, provider, vm_name):
         lambda: rest_api.collections.templates.find_by(name=vm_name).subcount != 0,
         num_sec=700, delay=15)
     return rest_api.collections.templates.get(name=vm_name)
+
+
+def arbitration_settings(request, rest_api, num=2):
```

Comment:
Possibly. This is how Taras structured it when he was creating the tests. If you look closely, you see it uses request, so this should be used similarly like fixtures, you can import this and then call it from inside a fixture, this was used because sometimes function scope was required and sometimes module scope was enough ...